### PR TITLE
Update wk8

### PR DESCRIPTION
**Pull request by BartBucknill**
- project challenge extended to include choice between implementing using promises (introduced in wk7) or using express middleware for session management.
- research afternoon time replaced by extended project time to allow for focused research (technical spikes) on the above.

### CODE REVIEW

**Comment by astroash** [comment]
Relates #784

@shiryz & @mantagen no need for this to be merged right now. This is the timetable London is running with for week 8. @BartBucknill you can share this PR with FAC12 on your week so they have the timetable 😄

File: coursebook/week-8/README.md [hunk 1]
```diff
@@ -22,29 +22,20 @@ Schedule
...
-
-- 15.00 - 17:00
--- [Research Afternoon](./research-afternoon.md)
+- 14:00 - 14:30
+-- [Introduce Project](./project.md) and assign teams
 
-- 17.00 - 18:00
--- Review and Present research topics
+- 14:30 - 18:00
+-- Project planning and technical spikes
```
Comment (des-des) [inline coursebook/week-8/README.md]: Can we add a (short) section called tech spike in project file and link it here

Comment (BartBucknill) [reply coursebook/week-8/README.md]: done, I just changed the name of what was 'tips' to 'technical spikes'

File: coursebook/week-8/project.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Previous cohorts have built apps where users can share, for example, blog posts,
 - Use Heroku or a similar service to host the app and the database.
 - Try to include ES6 syntax on the server.
 
-### Tips
+### Technical Spikes
 
 **Promises**:
   - In order to get promises back from queries to your PostgresSQL database, you might consider using the [pg-promise](https://www.npmjs.com/package/pg-promise) module.
```
Comment (astroash) [inline coursebook/week-8/project.md]: Do you think it's work suggesting using `cookie-session` middleware for the session management challenge too?

Comment (BartBucknill) [reply coursebook/week-8/project.md]: @astroash cookie-session is included, did you mean something else?

Comment (astroash) [reply coursebook/week-8/project.md]: @bart, literally no idea what I was talking about. I could swear lines 35+ were hiding from me last time.

**Comment by eliascodes** [comment]
@shiryz @mantagen @ibraghada could you let me know whether your week 8s followed this or a different schedule so we can understand whether or not to merge it?

**Comment by shiryz** [comment]
@eliascodes we did research as it was

**Comment by eliascodes** [comment]
@ibraghada could you update on which version of the schedule you followed?

**Comment by eliascodes** [comment]
Closing since this was only introduced in one campus and isn't part of the recommended changes for the next iteration. Thanks for your efforts everyone, it was a useful trial!

**Pull request closed without merging**